.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+# This CircleCI workflow builds our documentation into HTML pages.
+# The generated pages are viewable as artifacts in CircleCI.
+# CircleCI can be run on PRs from forks & allows viewing stored artifacts
+# without download.
```

Comment:
Do you want to call out the PR-from-fork bit is not possible from ADO?

Reply:
done